packages/components/src/pages/VsCodeExtension.vue

```diff
@@ -934,11 +934,9 @@ export default {
       ];
 
       events.forEach((e) => {
-        const eventName = e.codeObject.prettyName;
-
         if (
-          eventName.startsWith('GET') &&
-          mediaFileExtensions.some((ext) => eventName.endsWith(ext))
+          e.requestMethod === 'GET' &&
+          mediaFileExtensions.some((ext) => e.requestPath.endsWith(ext))
```

Comment:
`requestPath` can be destructured to avoid invoking the accessor for each media file extension.

Reply:
Done https://github.com/applandinc/appmap-js/pull/429/commits/41369ecb8ac787c473019980f31cdadfb920fde5